gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project-wide Gradle settings.
-VERSION_NAME=5.5.0-SNAPSHOT
+VERSION_NAME=5.5.0
```

Comment:
does I get right that you're gonna make one more PR, where this field will changed to 
`VERSION_NAME=5.6.0-SNAPSHOT`?
if so `README.md` changes should be done in the next PR, wdyt?

Reply:
I'm keeping it as `5.5.0` so that when I target this branch when I do the Github release/tag, CI sees the `5.5.0`. Then I make another commit on this branch to change it to `5.6.0` before actually merging this pr into to this repo's main branch.

Reply:
I see. I've added `TAG` directly at release branch 👍